app/services/core_data_connector/import/instances.rb

```diff
@@ -0,0 +1,116 @@
+module CoreDataConnector
+  module Import
+    class Instances < Base
+      def cleanup
+        super
+
+        execute <<-SQL.squish
+          WITH update_instances AS (
+            UPDATE
+            core_data_connector_instances
+              SET z_instance_id = NULL
+          )
+
+          UPDATE
+          core_data_connector_names
+            SET z_source_id = NULL,
+                z_source_type = NULL
+        SQL
+      end
+
+      def load
+        super
+
+        execute <<-SQL.squish
+          UPDATE core_data_connector_instances instances
```

Comment:
I know we talked about not updating names, but now that we're not attempting to share names during the import process, would it make sense to update names during import?

Reply:
I think it could cause conflicts if the user connects other records to that name after the initial import. Then if the import process is run again and changes the name, it will affect the other records too.

Reply:
Hmm, that's a good point. 👍